db/migrations/20240514205117-trim-tabs-from-plan-names.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    async up(q, Sequelize) {
+        // Fix plan names that would be duplicates once TRIM()-ed
+        const remove_dup_tabs = `
+            WITH dups AS (  
+                SELECT a.name, b.id  
+                FROM remediations a  
+                JOIN (  
+                    SELECT TRIM(TRIM('\\\\t' FROM name)) AS trimmed_name, id, tenant_org_id  
```

Comment:
Just wondering whether a string replace would together with a trim could be a better choice, to cover cases where tabs are in-between.

Reply:
Tabs in the middle are actually allowed, as are lots of other characters - I think the idea was to allow for other codesets.